src/jwt_util.rs

```diff
@@ -43,10 +43,14 @@ async fn get_rs256_key(client: &Client) -> EsiResult<String> {
 }
 
 /// Decode and validate the JWT token
-fn validate(token: &str, client_id: &str, decoding_key: &DecodingKey) -> Result<TokenClaims, EsiError> {
+fn validate(token: &str, client_id: Option<&str>, decoding_key: &DecodingKey) -> Result<TokenClaims, EsiError> {
     let mut validations = Validation::new(Algorithm::RS256);
     validations.required_spec_claims = vec![String::from("sub")].into_iter().collect();
-    validations.set_audience(&[client_id, "EVE Online"]);
+    let mut aud = vec!["EVE Online"];
+    if let Some(id) = client_id {
```

Comment:
Do you spot a use-case where the library would be validating a JWT but wouldn't have a client ID?

Reply:
Not really actually. A wrong assumption on my part. I will correct that.

Reply:
All good!